frappe/public/js/frappe/file_uploader/index.js

```diff
@@ -19,7 +19,7 @@ export default class FileUploader {
 		} else {
 			this.wrapper = wrapper.get ? wrapper.get(0) : wrapper;
 		}
-
+		let private_only = (parseInt(frappe.boot.sysdefaults.is_private_only)) ? true:false;
```

Comment:
I am just wondering, we have checked the box (Make all attachments private) and disabled it, which makes adding a checkbox in System Settings obsolete. Even if we uncheck it, nothing will change.